### DISCUSSION TITLE
How should the "Readwise Items To Review" note-tool be used?

### DISCUSSION THREAD

**Discussion by jacobholm** [BryanJenksCommunity/FAQ#180] [Q&A] [answered]
I am unsure of how this is intended to be used. What is your workflow?

Say I have 5 new, unprocessed books (or articles or whatever) in my readwise folder. I go into the "Readwise Items To Review" note to see them. Then what? Do I create a new book note for each of them, copy over the contents and then update the "lastchecked" attribute?

**Comment by tallguyjenks** [comment, marked as answer]
thats pretty much how i was using it.

My problem is that i have a lot of readwise inputs i haven't processed yet and i want to process them starting from the oldest thing moving forward, so i use that note to simply tell me whats left to process and then when i process the oldest thing on the list then i update lastchecked which says that the items it will show have to be older than the date in the title of the notes.

in looking at it, for some reason the note is no longer working for me with the lastchecked variable so I've modified it to be hard coded. same thing just no variable usage.

![image](https://github.com/BryanJenksCommunity/FAQ/assets/29872822/012b792f-c884-49c5-ae00-255f2920e421)

![image](https://github.com/BryanJenksCommunity/FAQ/assets/29872822/94218911-5b2a-4d2c-bbbd-ce81c37d8a9b)


so if i finished processing `; 2023-02-18 Quick Passages` then i'd update the line:

`WHERE file.day > date("2023-01-01")`

to 

`WHERE file.day > date("2023-02-18")`

which corresponds to the item i just finished. because now every date in the file names that does not have a date more recent that `2023-02-18` will be excluded. this is because I'm working from oldest to newest on these inputs. some exclusions are added (you can see them in the screen shots) because i simply processed them already ahead of others and don't want them to appear in the query output.

ultimately this note is just telling me what i still need to process from readwise as readwise creates the new notes with the default tags indicating things are unprocessed i.e. `#📥️/📰/🟥️` and i wont know which I've already processed unless i keep track in this way.

**Reply by jacobholm** [reply]
Nice, thanks for the thorough answer 👍
